app/services/github_issues_author.rb

```diff
@@ -0,0 +1,55 @@
+class GithubIssuesAuthor
+  attr_reader :existing_authors_ids
+
+  ALLOWED_ACTIONS = %w[opened edited deleted].freeze
+
+  def initialize(author_ids = [])
+    @existing_authors_ids = author_ids
```

Comment:
I'm injecting the already existing issue to avoid creating the same issue again, and to avoid an extra query to find if the issue exists, i know you can do it with "find_or_create_by" but this approach made more sense to me.

Reply:
Actually there is absolutely no need for this, i have added an "issue_id" to the author table to integrate with github, and a simple check for uniqueness is all that is required in my humble opinion what do you think ?